x-pack/plugins/uptime/public/components/queries/filter_bar/filter_bar.tsx

```diff
@@ -126,8 +126,8 @@ export const FilterBar = ({
         {
           type: 'field_value_selection',
           field: 'monitor.type',
-          name: i18n.translate('xpack.uptime.filterBar.options.typeLabel', {
```

Comment:
I don't know that we should change the name of the key at this point because it'll probably break our translations, and I don't know if/when they will do another extraction for `6.7.0`/`7.0`. We may need to create a follow-up issue for `7.1.0` to update this key's name.

Reply:
After speaking with Localization folks, this change is fine. @andrewvc you were right in that we have to update the key with the value in this case.